CMakeLists.txt

```diff
@@ -62,8 +62,18 @@ macro(qsk_setup_build)
 endmacro()
 
 macro(qsk_setup_install)
-    set(QSK_INSTALL_HEADERS include)
-    set(QSK_INSTALL_LIBS lib)
+
+    # package vars
+    set(PACKAGE_NAME      ${PROJECT_NAME})
+    set(PACKAGE_VERSION   ${CMAKE_PROJECT_VERSION})
+    set(PACKAGE_NAMESPACE ${PROJECT_NAME}::)
+    set(PACKAGE_LOCATION  ${CMAKE_INSTALL_LIBDIR}/cmake/${PROJECT_NAME})
+
+    # install paths for headers and libraries
+    string(TOLOWER "${PROJECT_NAME}" PACKAGE_DIR)
+    set(QSK_INSTALL_HEADERS ${CMAKE_INSTALL_INCLUDEDIR}/${PACKAGE_DIR})
+    set(QSK_INSTALL_LIBS    ${CMAKE_INSTALL_LIBDIR}/${PACKAGE_DIR})
+
```

Comment:
@uwerat @lgtmak the entire naming/packaging has not yet been discussed/defined

|CMake Variable|Value|Description|
|-|-|-|
|PROJECT_NAME|QSkinny|The QSkinny framework project's name|
|PACKAGE_NAME|QSkinny|The intention was to have `QSkinny` as the human readable package name|
|PACKAGE_NAMESPACE|QSkinny::|This is the namespace/prefix a consumer needs to address QSkinny package components, targets, etc. e.g. `QSkinny::QSkinny` for the shared (core) library or `QSkinny::svg2qvg`. My intention with `Qsk` was not to clutter the CMake files with another `QSkinny` expression and to emphasize the difference between project, target, package name and namespace name as well as being e.g. the C++ namespace qsk|

Reply:
Addressed in #358